src/components/theme/View/View.jsx

```diff
@@ -215,10 +202,14 @@ export default class View extends Component {
       <div id="view">
         <Helmet
           bodyAttributes={{
-            class: `view-${viewName.toLowerCase()}`,
+            class: RenderedView.displayName
+              ? `view-${RenderedView.displayName.toLowerCase()}`
+              : 'view-undefined',
```

Comment:
maybe `view-default` or something else would look better? `undefined` sounds a bit too javascripty to me, like an error